lib/widgets/user_profile.dart

```diff
@@ -5,283 +5,382 @@ import 'package:intl/intl.dart';
-                          style: theme.textTheme.bodyText1,
-                        ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(bottom: 8),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Icon(
+                      Icons.cake,
+                      size: 13,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.only(left: 4.0),
+                      child: Text(
+                        DateFormat('MMM dd, yyyy').format(userView.published),
+                        style: theme.textTheme.bodyText1,
                       ),
-                    ],
-                  ),
+                    ),
+                  ],
                 ),
-                Expanded(child: tabs())
-              ],
+              ),
+              // Expanded(child: tabs())
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class _AboutTab extends HookWidget {
+  final UserDetails userDetails;
+
+  const _AboutTab(this.userDetails);
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final instanceUrl = userDetails.user.instanceUrl;
+
+    final accStore = useAccountsStore();
+
+    final isOwnedAccount = accStore.loggedInInstances.contains(instanceUrl) &&
+        accStore.tokens[instanceUrl].containsKey(userDetails.user.name);
+
+    const wallPadding = EdgeInsets.symmetric(horizontal: 15);
+
+    final divider = Padding(
+      padding: EdgeInsets.symmetric(
+          horizontal: wallPadding.horizontal / 2, vertical: 10),
+      child: Divider(),
+    );
+
+    return ListView(
+      padding: EdgeInsets.symmetric(vertical: 20),
+      children: [
+        if (isOwnedAccount)
+          Center(
+            child: FlatButton.icon(
+              icon: Icon(Icons.edit),
+              label: Text('edit profile'),
+              onPressed: () {}, // TODO: go to account editing
             ),
           ),
+        if (userDetails.user.bio != null) ...[
+          Padding(
+              padding: wallPadding,
+              child:
+                  MarkdownText(userDetails.user.bio, instanceUrl: instanceUrl)),
+          divider,
         ],
-      ),
+        if (userDetails.moderates.isNotEmpty) ...[
+          Padding(
+            padding: wallPadding,
+            child: Text('Moderates', style: theme.textTheme.subtitle2),
+          ),
+          for (final comm in userDetails.moderates)
+            ListTile(
+              dense: true,
+              title: Text('!${comm.communityName}'),
+              onTap: () =>
+                  goToCommunity.byId(context, instanceUrl, comm.communityId),
+            ),
+          divider
+        ],
+        Padding(
+          padding: wallPadding,
+          child: Text('Subscribed', style: theme.textTheme.subtitle2),
+        ),
```

Comment:
section titles should be based on the ones in [instance page](https://github.com/krawieck/lemmur/blob/1eb3c9e937ec042f832b55b9cb9f886ca1b1c4f7/lib/pages/instance.dart#L347-L354), centered and  based on headline6 with fontSize set to